code/modules/mob/living/basic/lavaland/watcher/watcher_overwatch.dm

```diff
@@ -0,0 +1,136 @@
+/**
+ * Automatically shoot at a target if they do anything while this is active on them.
+ */
+/datum/action/cooldown/watcher_overwatch
```

Comment:
Subtype /datum/action/cooldown/mob_cooldown just for consistency

(This lets it be used with admin verb `add_mob_ability`)

Reply:
`mob_cooldown` sets a handful of things I don't want to set and then just need to unset again, with no other functionality than the admin menu.
I don't _want_ it to share a cooldown with the other ability.

Out of interest, why does the admin verb only check `mob_cooldown` subtypes rather than just all `action/cooldowns`?

Reply:
I guess including _every_ cooldown action would include a bunch of stuff which couldn't possibly work instead of just "some" stuff which can't possibly work. What a pain.